src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java

```diff
@@ -1907,7 +1907,9 @@ else if (clickedBlock != null &&
                 if (material.isLegacy()) continue;
                 if (material.name().endsWith("_SPAWN_EGG"))
                     spawn_eggs.add(material);
-                else if (material.name().endsWith("_DYE"))
+                else if (material.name().endsWith("_DYE")
+                		|| material.equals(Material.GLOW_INK_SAC)
+                		|| material.equals(Material.INK_SAC))
```

Comment:
Should add single special cases outside of the loop, no need to check every single time around. Also, you have mixed tabs and spaces.

Reply:
Ah, I see what you're talking about, I appreciate the heads up.  This commit should address both issues.